src/sonic-config-engine/config_samples.py

```diff
@@ -10,6 +10,34 @@
 else:
     UNICODE_TYPE = unicode
 
+
+# The new enhancements to this file provides capabilities to generate l1 & l3
```

Comment:
This comment looks more like PR description. I think this should be removed.

Reply:
Also, it will require the comment update again once someone add an additional template..

Reply:
Thanks. Fixed the comments and uploaded the new change.

Reply:
The tests for both L1 & L3 passed as belows. 

test_l1_ports_template (tests.test_j2files.TestJ2Files) ... ok
test_l3_ports_template (tests.test_j2files.TestJ2Files) ... ok


New failures are seen in unrleated area.